### ISSUE TITLE
Gallery not working

### ISSUE THREAD

**Issue by darkstar75** [splittingred/Gallery#185]
After installing latest Gallery (1.4.0) on Modx revo 2.2.0, creating gallery, uploading images and calling them with [[!Gallery? &album=`5`]] I get only names of images and no thumbs on web page. clicking names doesn't show photos.

gallery page url: http://67.225.136.222/~zdrava/index.php?id=4

**Comment by chris-dempsey** [comment]
Should the call not be:

[[!Gallery? &album=`5`]]

Assuming the name of your Album is 5 and that's not the ID assigned to it by the component in MODX?

**Comment by Mark-H** [comment]
You either pass the name of an album or the ID of the album to the &album property. If it sees a string it uses it as the name, if it sees a number it uses it as the ID.

**Comment by darkstar75** [comment]
Thanx all. I eventually decided Revolution is too big for the size of site I am making, so I switched to good old Evolution where things work predictably.

**Comment by chris-dempsey** [comment]
Looking at my reply I see the backticks I typed are not displayed visibly around the 5.  When replying to the original post I thought darkstar76 had missed the backticks in the snippet call.  However, looking more closely (actually, changing the angle of my screen) I see that the backticks are rendered as a <code> block styled with a faint grey outline.

Interestingly, I tested the snippet specifying an album ID with and without backticks and both work (Gallery 1.4.0-rc1 and Revo 2.2.0-pl2).  I thought backticks around the property value were a requirement regardless of whether it would be treated by the snippet as a string or integer.
